README.md

```diff
@@ -71,7 +71,22 @@ See the JSDoc markdown documentation for the returned `balena` object in [DOCUME
 Support
 -------
 
-If you're having any problem, please [raise an issue](https://github.com/balena-io/balena-sdk/issues/new) on GitHub and the balena team will be happy to help.
+If you face any issues, please [raise an issue](https://github.com/balena-io/balena-sdk/issues/new) on GitHub and the balena team will be happy to help.
+
+Deprecation policy
+------------------
+
+The `balena-sdk` uses [semver versioning](https://semver.org/), with the
+concepts of major, minor and patch version releases.
+
+The latest release of the previous major version of the balena-sdk will be
+supported for one year from the date when the next major version is released.
+For example, balena-sdk v12.33.4 (the latest v12 release) will be supported for
+one year from the date when v13.0.0 is released.
```

Comment:
NIT in this paragraph and the previous one:
* Some occurrences of `balena-sdk` are enclosed in backticks, others are not. If using a dash, I suggest always enclosing in backticks: `balena-sdk` _vs._ "balena SDK".
* Use of "the" before "balena-sdk". The way I read it:
  * _"The `balena-sdk` uses semver..."_ (bad)
  * _"The balena SDK uses semver..."_ (good)
  * _"`balena-sdk` uses semver"_ (good)

When written with a dash, and even more so if enclosed in backticks, `balena-sdk` sounds to me like the name of the package. Replace it with "Microsoft" to see how it sounds:

  * _"The Microsoft uses semver..."_ (bad)
  * _"The Microsoft SDK uses semver..."_ (good)
  * _"Microsoft uses semver"_ (good)

Reply:
Makes sense @pdcastro :+1:  even though I don't feel that
> The `balena-sdk`

is that bad.

I will go with
> the balena SDK

approach since that's more consistent with the rest of the README.